src/models/language.rs

```diff
@@ -98,11 +100,20 @@ impl Default for PiranhaLanguage {
 
 impl From<&str> for PiranhaLanguage {
   fn from(language: &str) -> Self {
+    PiranhaLanguage::from_str(language).unwrap()
```

Comment:
Ok. So the deal is that earlier I I was implementing this `From<&str>` trait, and was simply panic-ing when language was not supported. 

Upon adding clap (Command line argument parsing) support for `language: PiranhaLanguage`, I needed to implement the `trait std::str::FromStr` (because `clap` requires me to do , since invokes it to parse the input string into `PiranhaLanguage`). Note, `FromStr` has a return type `Result<X,Err>` while `From` has a return type `X`. 

So I moved the implementation of `From` into `FromStr`. 
Then I call `FromStr` with unsafe unwrap (inside `From`). When the language name is not supported we just panic.

Alternatively, I could remove the implemetation of `From` trait  and update all usages to `FromStr`. But this will involve handling `Result<PiranhaLanguage, Err>` at each callee of `FromStr`

Reply:
Shouldn't we return a graceful error to the user and a CLI help description of how to use the tool when they pass an unknown `--language`? It sounds from the above that we would just panic in that case, which makes sense when building from the python/rust API, but probably not graceful enough for CLI input...

Reply:
That's pretty cool, actually 🙂